app/models/service_plan.rb

```diff
@@ -1,4 +1,5 @@
 require "archived_concern"
+require_relative "../../lib/infrastructure/external_api_broker"
```

Comment:
`./lib` is on the LOAD_PATH so you shouldn't need the require_relative

Reply:
It's probably not auto-loading because the constants are not in the `Infrastructure` namespace

Reply:
Yeah we can talk about the directory naming structure, this was just something I've used in the past, and it happens to avoid something like `lib/services` since we have the name "service" in a bunch of actual objects, haha!